sections/infrastructure.tex

```diff
@@ -1,30 +1,61 @@
-\item \href{https://readthedocs.org/}{Read the Docs}: This is where our \href{http://docs.sunpy.org/en/stable/}{online documentation} is hosted, they also build the documentation in the process.
-\item \href{https://codecov.io/}{Codecov}: This aggregates the results from each test suite to monitor the amount of lines covered by the library's test suite.
+\itemsep0em
+\item 1.0.0 is an LTS release, released November 2019.
+\item 1.1.0 is a short support release, released May 2020.
+\item 1.0.1 and 1.1.1 are bug-fix releases, released June 2020, to fix a current bug and back-port the fix to the LTS version.
+\item 2.0.0 is an LTS release, released November 2020.
 \end{itemize}
 
+One benefit of users using an LTS release is that it will provide warnings for any functionality that will be changed or removed for the next LTS release.
+Where as changes to the code that could break existing scripts, will not happen until the next LTS release but could happen on the non-LTS releases.
+This provides a stable base for long term \sunpy users.
+
+\subsection{Continuous Integration - Nabil}
+The \sunpyproj follows common practices and makes extensive use of continuous integration
+which refers to the practice of automated testing and code change inspection.  
+Any and all proposed code contributions to \sunpypkg trigger test suits to be run on a number of free services which integrate into the \github website. 
+These services provide the first review of all new code and therefore maintain the integrity of the package which makes it easier for new contributors to understand the impact of their changes.
+
+\sunpypkg's test suite can be broken down into three broad categories: offline, online and figure tests. 
+Offline test suite checks the majority of the codebase. 
+Online tests specifically checks code that contacts generalized data service providers such as the VSO \todo{(predefined else where?)} or JSOC as well as specific data providers like GOES or NOAA. 
+These tests depend on the availability of these online services. 
+Finally figure tests check code that generate plots and will generate a failure if the image has changed. 
+This enables high level functionality checks.
+
+\sunpy makes use of the following online services:
+
+\begin{itemize}
+\item \href{https://azure.microsoft.com/en-gb/services/devops/pipelines/}{Microsoft's Azure pipelines}: Runs the entire test suite on each operating system (Windows, Mac, Linux).
+\item \href{https://circleci.com}{circleci}: Runs the library's documentation, a special 32bit build and the figure tests.
+\item \href{https://travis-ci.org}{Travis CI}: Runs the entire test suite on a daily cadence.
+\item \href{https://codecov.io/}{Codecov}: Provides code coverage metrics.
+\end{itemize}
 
-\subsection{Package Template}
-Provided for affiliated packages
```

Comment:
Where does this go now?

Reply:
We will just add a sentence about it in the affiliated packages section.